src/main/java/org/irods/Serialize/ModifyPermissionsOperations.java

```diff
@@ -16,9 +18,19 @@ public class ModifyPermissionsOperations {
     @JsonProperty("acl")
     private String acl;
 
-    public ModifyPermissionsOperations(String entity_name, String acl) {
+    public ModifyPermissionsOperations(String entity_name, Permission acl) throws IrodsException {
         this.entity_name = entity_name;
-        this.acl = acl;
+        switch (acl) {
+            case NULL:
+            case READ:
+            case WRITE:
+            case OWN:
+                this.acl = acl.getValue();
+                break;
+            default:
+                throw new IrodsException("'" + acl.getValue() +
```

Comment:
Let's use `IllegalArgumentException` to help the dev understand what went wrong.

Reply:
Done.